examples/rl_sim_cpp/rl_sim.cc

```diff
@@ -602,7 +602,7 @@ std::string rl_sim::create_event_id()
   return oss.str();
 }
 
-rl_sim::rl_sim(boost::program_options::variables_map vm) : _options(std::move(vm)), _loop_kind(CB)
+rl_sim::rl_sim(const boost::program_options::variables_map& vm) : _options(vm), _loop_kind(CB)
```

Comment:
Out of scope for this PR... but would be good to get rid boost::program_options at some point since VW is almost free.